test/gpu/nvidia_test.go

```diff
@@ -83,6 +90,45 @@ var expectedDimsToMetrics = map[string][]string{
 	},
 }
 
+var expectedDimsToMetricsE2E = map[string][]string{
+	"ClusterName": {
+		containerMemTotal, containerMemUsed, containerPower, containerTemp, containerUtil, containerMemUtil,
+		podMemTotal, podMemUsed, podPower, podTemp, podUtil, podMemUtil,
+		nodeMemTotal, nodeMemUsed, nodePower, nodeTemp, nodeUtil, nodeMemUtil, podLimit, podTotal, podRequest, nodeCountTotal, nodeCountRequest, nodeCountLimit, clusterCountTotal, clusterCountRequest,
+	},
+	"ClusterName-Namespace": {
+		podMemTotal, podMemUsed, podPower, podTemp, podUtil, podMemUtil, podLimit, podTotal, podRequest,
+	},
+	//"ClusterName-Namespace-Service": {
+	//	podMemTotal, podMemUsed, podPower, podTemp, podUtil, podMemUtil,
+	//},
```

Comment:
Why is this commented out?

Reply:
this was copied over from expectedDimsToMetricsIntegTest which is in main branch